mshadow/base.h

```diff
@@ -279,7 +279,9 @@ struct DataType<float> {
   static const int kFlag = kFloat32;
   static const int kLanes = 1;
 #if MSHADOW_USE_CUDA
+#if (CUDA_VERSION >= 8000)
   static const cudaDataType_t kCudaFlag = CUDA_R_32F;
+#endif
```

Comment:
Do you still need this for cuda 7050?
I think there used to be a different enum.